packages/react-button/package.json

```diff
@@ -59,13 +59,13 @@
     "@uifabric/react-hooks": "^7.13.0",
     "@uifabric/set-version": "^7.0.22",
     "@uifabric/utilities": "^7.31.0",
-    "office-ui-fabric-react": "^7.137.1",
     "tslib": "^1.10.0"
   },
   "peerDependencies": {
     "@types/react": ">=16.8.0 <17.0.0",
     "@types/react-dom": ">=16.8.0 <17.0.0",
     "react": ">=16.8.0 <17.0.0",
-    "react-dom": ">=16.8.0 <17.0.0"
+    "react-dom": ">=16.8.0 <17.0.0",
+    "office-ui-fabric-react": ">=7.137.1 <8.0.0"
```

Comment:
could you add a new issue to track reverting this change later?

Reply:
#15028

Reply:
#15028